interface/src/app/tribe_client/geneset-search-bar.tpl.html

```diff
@@ -1,12 +1,11 @@
 <div>
   <form class="form-inline">
     <label>Organism:</label>
-    <select ng-disabled="true">
-      <option ng-selected="true" value="organism">{{organism}}</option>  
-    </select>
+    <i>{{$ctrl.organism}}</i>&nbsp;
+    <input ng-model="$ctrl.search.query" placeholder="Search text">
```

Comment:
When no MLModel is selected (*way* at the top of this page, so out of sight), this looks like a form to search for an **Organism** because `{{$ctrl.organism}}` (line 4 above) is blank. In addition, all of my searches were failing silently, so I couldn't figure out how to make this work.

I'd suggest adding a message at line 4 saying something like "please select a **Machine Learning Model** above" and disabling this input until an MLModel is selected. (Assuming that is required for this to work.)